config/ecs/config.go

```diff
@@ -0,0 +1,109 @@
+package ecs
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-tf-aws/config/common"
+)
+
+func init() {
+
+	config.Store.SetForResource("aws_ecs_cluster", config.Resource{
+		ExternalName: config.ExternalName{
+			// Note(turkenh): Seems like we have a case that breaks our
+			// assumption that "id" field always used to import resource and
+			// should be set as external-id. Here we could import the resource
+			// with "name" but id contains "arn".
+			// https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_cluster
+			ConfigureFunctionPath: common.PathExternalNameAsName,
+			OmittedFields: []string{
+				"name",
+			},
+		},
+		References: config.References{
+			"capacity_providers": config.Reference{
+				Type: "CapacityProvider",
+			},
+			"execute_command_configuration[*].kms_key_id": config.Reference{
+				Type:              "github.com/crossplane-contrib/provider-tf-aws/apis/kms/v1alpha1.Key",
+				RefFieldName:      "KmsKeyRef",
```

Comment:
The reason we explicitly include the full field name is to let users know that the reference is used to fill that specific field, otherwise from CRD API schema perspective there is nothing that tells you that ref field targets that value field. For more details https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-of-the-reference-field